src/api/spec/features/beta/webui/requests_spec.rb

```diff
@@ -86,7 +86,7 @@
         visit request_show_path(bs_request)
         click_button 'Accept'
 
-        expect(page).to have_text("Request #{bs_request.number} (accepted)")
```

Comment:
If you remove this `(accepted)` string, then you should add another expectation checking the presence of the accepted badge markup and class